src/lib/DB.js

```diff
@@ -54,12 +54,6 @@ class DB {
 
 export default DB;
 
-//DEPRECATE on v2.0.0
-export const sha256 = (text, encoding='base64') => {
-  console.log('DB.sha256 is deprecated. Please use sha256 from @klaudsol/commons/lib/Crypto instead.');
-  return sha256(text, encoding);
-}; 
-
```

Comment:
This is not backward compatible. Erasing this will break a lot of code that relies on `sha256`. Are you sure that you did not include `sha256` in your code somewhere else?